Shippo/Parcel.cs

```diff
@@ -1,40 +1,51 @@
 ﻿using System;
 using Newtonsoft.Json;
 
-namespace Shippo {
+namespace Shippo
+{
     [JsonObject (MemberSerialization.OptIn)]
-    public class Parcel : ShippoId {
+    public class Parcel : ShippoId
```

Comment:
Looks like you added new properties. In that case the members can be typed. Sorry didn't notice them earlier.